src/run.py

```diff
@@ -67,6 +90,8 @@ def run(config_filename, log_filename, log_to_stderr):
     network_config.load_network(config_filename)
 
     agents = initialize(network_config, config_filename)
+    if vis:
+        visualisation(network_config, logger)
```

Comment:
O ile się nie mylę to visualisation ma swoją pętlę z której program nie wyjdzie i nie przejdzie do CLI - można to rozwiązać robiąc w visualisation:
thread = threading.Thread(target=real_time_plot, args=(graph, eventDict))
thread.start()